

--- origsrc/fox-1.6.43/include/FXJPGIcon.h	2010-09-01 20:00:28.000000000 -0500
+++ src/fox-1.6.43/include/FXJPGIcon.h	2011-02-11 06:48:35.863961700 -0600
@@ -87,7 +87,7 @@ extern FXAPI bool fxloadJPG(FXStream& st
 /**
 * Save an JPEG (Joint Photographics Experts Group) file to a stream.
 */
-extern FXAPI bool fxsaveJPG(FXStream& store,const FXColor* data,FXint width,FXint height,FXint quality);
+extern FXAPI bool fxsaveJPG(FXStream& store,FXColor* data,FXint width,FXint height,FXint quality);
 
 }
 
--- origsrc/fox-1.6.43/include/FXJPGImage.h	2010-09-01 20:00:28.000000000 -0500
+++ src/fox-1.6.43/include/FXJPGImage.h	2011-02-11 06:48:43.312387800 -0600
@@ -86,7 +86,7 @@ extern FXAPI bool fxloadJPG(FXStream& st
 /**
 * Save an JPEG (Joint Photographics Experts Group) file to a stream.
 */
-extern FXAPI bool fxsaveJPG(FXStream& store,const FXColor* data,FXint width,FXint height,FXint quality);
+extern FXAPI bool fxsaveJPG(FXStream& store,FXColor* data,FXint width,FXint height,FXint quality);
 
 }
 
--- origsrc/fox-1.6.43/src/fxjpegio.cpp	2010-09-01 20:00:28.000000000 -0500
+++ src/fox-1.6.43/src/fxjpegio.cpp	2011-02-11 06:48:05.326215100 -0600
@@ -74,7 +74,7 @@ namespace FX {
 
 extern FXAPI bool fxcheckJPG(FXStream& store);
 extern FXAPI bool fxloadJPG(FXStream& store,FXColor*& data,FXint& width,FXint& height,FXint& quality);
-extern FXAPI bool fxsaveJPG(FXStream& store,const FXColor* data,FXint width,FXint height,FXint quality);
+extern FXAPI bool fxsaveJPG(FXStream& store,FXColor* data,FXint width,FXint height,FXint quality);
 
 
 #ifdef HAVE_JPEG_H
@@ -291,7 +291,7 @@ static void term_destination(j_compress_
 
 
 // Save a JPEG image
-bool fxsaveJPG(FXStream& store,const FXColor* data,FXint width,FXint height,FXint quality){
+bool fxsaveJPG(FXStream& store,FXColor* data,FXint width,FXint height,FXint quality){
   jpeg_compress_struct dstinfo;
   FOX_jpeg_error_mgr jerr;
   FOX_jpeg_dest_mgr dst;
@@ -398,7 +398,7 @@ bool fxloadJPG(FXStream&,FXColor*& data,
 
 
 // Stub routine
-bool fxsaveJPG(FXStream&,const FXColor*,FXint,FXint,FXint){
+bool fxsaveJPG(FXStream&,FXColor*,FXint,FXint,FXint){
   return false;
   }
 
